.changeset/strong-goats-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+"@azure-tools/cadl-ranch-specs": minor
```

Comment:
Do we want to start bunping minor? Maybe just patch for now still

Reply:
oh good point, let me fix that